src/components/NavBar.js

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
 const NavBarContainer = styled.div`
   width: 100%;
   height: 4rem;
-  background: steelblue;
+  background: #26a69a;
   display: flex;
   justify-content: space-around;
   align-items: center;
   position: relative;
-  box-shadow: 0 0.7rem 1.2rem rgba(0, 0, 0, 0.09),
-    0 0.4rem 0.4rem rgba(0, 0, 0, 0.16);
+  box-shadow: ${props =>
```

Comment:
Also as discussed you could probably just apply the box shadow to the settings menu instead, removing the need for a props-based solution. It's also quite a harsh shadow, drop the opacity down a notch while you're at it.